tests/optimizations/byte_array_test.toit

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
```

Comment:
Again this shows that the tests may need a new 0BSD copyright header.

Reply:
I haven't changed the copyrights of any tests yet.
Once the migration is done, I will change them.